docs/internet-identity-spec.adoc

```diff
@@ -132,9 +132,9 @@ where `msg` is a value with of type
 where
 
 * the `sessionPublicKey` contains the public key of the session key pair.
-* the `maxTimeToLive`, if present, indicates the desired time span until the requested delegation should expire. The Identity Provider frontend is free to set an earlier expiry time, but should not create a larger.
+* the `maxTimeToLive`, if present, indicates the desired time span (in nanoseconds) until the requested delegation should expire. The Identity Provider frontend is free to set an earlier expiry time, but should not create a larger.
```

Comment:
Wasn't sure about this one, but other time spans in the doc are expressed as nanos.